### ISSUE TITLE
Final issue about the URL

### ISSUE THREAD

**Issue by tleylan** [iBicha/youtube-dl-server#13]
I get all the JSON data returned (I'm parsing it on my service) and I can easily see the id, the title, a ton of other data and an element named webpage_url but that is just the url I passed in with the video code. I see a channel_url and that plays in a browser but it returns the video playing within the channel page.

Is it hidden in one of the format elements? It played on my headset fine when I used your service Heroku service.

Any help is appreciated.

**Comment by iBicha** [comment]
What's your query url, and the result?

**Comment by tleylan** [comment]
I've changed the name of my server but basically I send this string

https://myserver.com/url?url=OZJhhY5h9Hc&cli=yt-dlp

I added an endpoint named url so a) I didn't change yours and b) I can parse the mass of JSON that is returned and extract only a url. It is returning JSON like { "url": "the_url"} Using Postman I receive the JSON and in my headset I can parse that JSON and extract just the url. But I don't know which JSON field is supposed to contain the url to use.

I copied the code from your app.get('/v1/video', async (req, res) I could fetch all the JSON if it would help you but it is what you return from the this one.
